Bixal/move.mil
# Update drupal core

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Updates drupal core to 8.6.9
- Updates outdated drupal modules
- Updates outdated theme JS libraries
- Updates outdated React tools JS libraries

## Testing

To verify the changes proposed in this pull request…

1. Build theme: npm install and npm run build
1. Build tools: npm install and npm run build
1. On root: composer install and clear caches
